src/data/resolvers/motions.ts

```diff
@@ -20,6 +20,52 @@ import {
 
 import { ProcessedEvent } from './colonyActions';
 
+const getMotionEvents = async (
+  votingReputationClient: ExtensionClient,
+  motionId: string,
+) => {
+  const motionStakedLogs = await getLogs(
+    votingReputationClient,
+    // @TODO Add missing types to colonyjs
+    // @ts-ignore
+    votingReputationClient.filters.MotionStaked(motionId, null, null, null),
+  );
+
+  const parsedMotionEvents = await Promise.all(
+    [...motionStakedLogs].map(async (log) => {
+      const parsedLog = votingReputationClient.interface.parseLog(log);
+      const { address, blockHash, blockNumber, transactionHash } = log;
+      const {
+        name,
+        values: { amount, ...rest },
+      } = parsedLog;
+      const stakeAmount = name === 'MotionStaked' ? amount : null;
```

Comment:
Please use the type in here

```suggestion
      const stakeAmount = name === ColonyAndExtensionsEvents.MotionStaked ? amount : null;
```